Understanding ADR/README.md

```diff
@@ -0,0 +1,244 @@
+# Introduction
+
+Software design plays crucial design for success of a software project, especially in large teams working on big or old codebase. Software design includes making important decisions about the structure, functionalty, and architeture of the software system. However, making these decisions can be challenging, especially in a collaborative environment where misunderstandings and miscommunication can occur.
+In this micro-byte we will understand:
+
+- What are ADR?
```

Comment:
just a simple typo here , it should be "What is ADR"

Reply:
Thank you, I will update it.